src/app/BUILD.gn

```diff
@@ -473,6 +474,7 @@ static_library("app") {
     "${chip_root}/src/messaging",
     "${chip_root}/src/protocols/interaction_model",
     "${chip_root}/src/system",
+    "${chip_root}/third_party/inipp",
```

Comment:
Do not add 3rd party deps to libCHIPDataModel without justification

Reply:
inipp does not seem to make sense for embedded devices